lm75bd/lm75bd.c

```diff
@@ -5,6 +5,7 @@
 
 #include <stdint.h>
 #include <string.h>
+#include <stdio.h>
```

Comment:
I don't think this include is necessary?

Reply:
yes, good catch